site-lib-src/bigarray/META.in

```diff
@@ -1,6 +1,6 @@
 # Specifications for the "bigarray" library:
 requires = "unix"
-version = "[distributed with Ocaml]"
+version = "[distributed with OCaml]"
```

Comment:
Again, I'd let these be - OCaml 5.x doesn't install any of these `META` files, and given that these strings do get matched on, it just doesn't seem worth the risk!

Reply:
Reverted.